src/app/welcome/get.controller.js

```diff
@@ -1,6 +1,7 @@
 const logger = require('../../common/utils/logger')(__filename);
+const homePageMessage = process.env.HOMEPAGE_MESSAGE;
```

Comment:
Hey Sarita, the other environment variables are pulled in `./src/common/config/index.js` I think it would be good, if possible to do the same with HOMEPAGE_MESSAGE and then import config.

Reply:
done